setup.py

```diff
@@ -42,10 +42,10 @@
   ],
 
   install_requires=[
-    'click>=5.0,<6.0',
-    'PyYAML>=3.12,<3.13',
-    'requests>=2.18.0,<2.19',
-    'screepsapi>=0.4.3'
+    'click>=5.0',
+    'PyYAML>=3.12',
```

Comment:
Careful with this, PyYAML 4 is in beta and once it gets released this will upgrade without an upper limit. I don't know how badly 4 will break BC with 3 (if at all), but yeah . . .

Reply:
Edited to keep on same major version, I'm used to node where it does semver, and doesn't upgrade major versions by default.